canopy/saplingjs/src/index.ts

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+export { http, submitBatch } from './submitter';
```

Comment:
I don't think we should expose `http`.  This should just be a helper function for other things in this library.

Reply:
no longer exported